mysql-test/suite/innodb/t/instant_alter_bugs.test

```diff
@@ -565,4 +565,13 @@ ALTER TABLE t1 DROP COLUMN a;
 ALTER TABLE t1 DROP COLUMN c;
 DROP TABLE t1;
 
+--echo #
+--echo #  MDEV-35122  Incorrect NULL value handling for instantly
+--echo #                 dropped BLOB columns
+--echo #
+CREATE TABLE t1 (c1 INT, c2 BLOB) ROW_FORMAT=REDUNDANT,ENGINE=InnoDB;
+ALTER TABLE t1 DROP c2;
```

Comment:
Could we also add and drop a `c3 BLOB NOT NULL` column to exercise that code path as well?